src/OneExplorer/ConfigObject.ts

```diff
@@ -172,7 +179,10 @@ export class ConfigObj {
       return null;
     }
 
-    return new ConfigObj(uri, obj as Cfg);
+    const cfgObj = new ConfigObj(uri, obj as Cfg);
+    cfgObj.init();
+
+    return cfgObj;
```

Comment:
It creates `ConfigObj` whenever `createConfigObj` function is called. Is this your intention?

Reply:
Yes, it's to ease the process of undefined uri handling.